obs-packaging/wait-obs.sh

```diff
@@ -92,22 +106,25 @@ EOT
 }
 
 main() {
-	run_in_docker  $@
+	run_in_docker $@
 	local no_wait_publish="false"
 	case "${1:-}" in
-		"-h"|"--help")
-			usage "Help" 0
-			;;
-		--no-wait-publish)
-			no_wait_publish="true"
-			shift
-			;;
-		-*)
-			usage "Invalid option: ${1:-}" 1
-			;;
+	"-h" | "--help")
+		usage "Help" 0
+		;;
+	--no-wait-publish)
+		no_wait_publish="true"
+		shift
+		;;
+	-*)
+		usage "Invalid option: ${1:-}" 1
+		;;
 	esac
-	OBS_SUBPROJECT="${OBS_SUBPROJECT:-releases:x86_64:alpha}"
-	project="home:katacontainers:${OBS_SUBPROJECT}"
+	project=${1:-}
```

Comment:
Since you are using `project` as a global variable, please declare it at the beginning of the file.